examples/userapi/07_sparse_operations.ipynb

```diff
@@ -0,0 +1,965 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    coords = np.random.rand(npoint, 2)/2 + .25
</pre><p>Do you need to set a seed here? Otherwise this is going to be non-deterministic</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/2252/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>